### ISSUE TITLE
BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding

### ISSUE THREAD

**Issue by naoki-shigehisa** [naoki-shigehisa/paper#7]
## 0. 論文
[BERT: Pre-training of Deep Bidirectional Transformers for
Language Understanding](https://arxiv.org/pdf/1810.04805.pdf)

<img width="566" alt="スクリーンショット 2020-04-10 10 47 23" src="https://user-images.githubusercontent.com/43877096/78954959-aff93d80-7b18-11ea-925d-b794265685fa.png">

2019/5/24

## 1. どんなもの？
新たな言語表現モデルである、transformerによる双方向エンコーダー「BERT」を提案。

## 2. 先行研究と比べてどこがすごい？
全てのレイヤーで双方向のコンテキストを組み合わせて学習することで、ラベルのないテキストから深い双方向表現を事前学習できる。
様々な自然言語処理タスクに広く活用できる。

## 3. 技術や手法のキモはどこ？
Masked LMとNext Sentence Prediction (NSP)という2つのタスクによって事前学習を行う
- Masked LM
双方向の学習を同時に行うと間接的に自分自身がわかってしまうため、単純に双方向の学習を行うことはできない。文章中の単語の15%をランダムにマスクし、マスクされた単語を予測する。
- Next Sentence Prediction (NSP)
2つの文が連続しているものかどうかを予測する

<img width="841" alt="スクリーンショット 2020-04-10 11 21 28" src="https://user-images.githubusercontent.com/43877096/78956500-737c1080-7b1d-11ea-8ddf-e1b7035ca0a0.png">

## 4. どうやって有効だと検証した？
GLUE, SQuAD v1.1,SQuAD v2.0,SWAGなど11の自然言語処理タスクでSOTA
<img width="627" alt="スクリーンショット 2020-04-10 11 11 56" src="https://user-images.githubusercontent.com/43877096/78956083-29def600-7b1c-11ea-9e1c-5dac142d54aa.png">

## 6. 次に読むべき論文は？
XLNet: Generalized Autoregressive Pretraining for Language Understanding
